featurebyte/core/generic.py

```diff
@@ -148,13 +147,14 @@ def node_types_lineage(self) -> list[NodeType]:
         -------
         list[NodeType]
         """
+        # pylint: disable=no-member
```

Comment:
Is this pylint disable still needed with the pylint plugin?

Reply:
Good point, this can be removed.